enhancements/scheduling/scheduler-profiles.md

```diff
@@ -0,0 +1,208 @@
+
+### Non-Goals
+
+* configuration of individual scheduler plugins
+* platform support for custom schedulers or out-of-tree plugins
+
+## Proposal
+
+This enhancement proposes a new field in `schedulers.config.openshift.io/v1` named `profile`. This field
+specifies which of the following plugin configurations to enable on the default cluster scheduler.
+
+* `Default`: this will use the default scheduler profile as defined in the kube-scheduler's internal registry.
+This is the equivalent of currently passing no config to the scheduler, and in theory may change if the scheduler's
+list of default ("in-tree") plugins changes.
+  * Currently, this equates to a "Low Utilization" profile, with `NodeResourcesLeastAllocated` enabled
+
+* `HighUtilization`: this will disable `NodeResourcesLeastAllocated` and enable `NodeResourcesMostAllocated` to enable
+bin-packing of workloads onto nodes.
+
+* `NoScoring`: this is a "low latency" profile which strives for the quickest scheduling cycle by disabling all score plugins. 
+Score plugins are inherently non-critical so their exclusion should be safe and provide decreased scheduling time.
+
+### User Stories [optional]
+
+#### Story 1
+
+As a sysadmin, I want my workloads to be packed tightly onto nodes rather than the default behavior of spreading workloads 
+out among nodes. I enable the `HighUtilization` profile.
+
+#### Story 2
+
+As a sysadmin, I want to schedule my workloads as quickly as possible without regard for any soft preference of specific nodes. 
+I enable the `NoScoring` profile to achieve this.
+
+### Implementation Details/Notes/Constraints [optional]
+
+The new field will be set in the operator config:
+```yaml
+kind: Scheduler
+apiVersion: config.openshift.io/v1
+spec:
+  ...
+  profile: Default | HighUtilization | NoScoring | ...
+```
+
+The available values will be constrained to those defined above by API validation, but there is the possibility of adding more 
+options if significant use cases arise for different configurations.
+
+### Risks and Mitigations
+
+* Security and stability risks should be minimal, as this method of configuration as actually more restrictive than what is 
+currently supported for the scheduler
+* However, due to that there is a risk that this regresses from the current depth of configuration supported for customers with 
+very specific needs (based on metrics gathered, there are likely very few clusters that would be impacted in this way)
+* Transition from the old Policy API will provided by supporting the new Profile field alongside the old API (at least until Policy is 
+removed upstream, currently targeting k8s 1.23) with the restriction that only one may be set at a time.
+
+## Design Details
+
+The raw scheduler config corresponding to the set value will be merged into the existing postbootstrap component config 
+for the scheduler by the operator (except in the case of `Default`, where no raw config is necessary).
```

Comment:
I guess we'll keep a config map with pre-defined profiles, similarly to what we do with audit profiles in https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/v4.1.0/kube-apiserver/audit-policies-cm.yaml

Reply:
Yeah exactly, a bindata representation of each would imo be the easiest to maintain in the repo (though could get cumbersome if we add many profiles)